apps/portals/b2ai.standards/src/components/ChallengesCardDeck.tsx

```diff
@@ -1,66 +1,183 @@
-    ctaButtonText: 'Explore Standards',
-    ctaButtonURL: createExplorePageLink(CLINICAL_CARE_CHALLENGE_FILTERED_QUERY),
-    headerImage: <img src={healthSvg} />,
-  },
-]
+  if (!sql) {
+    sql = `SELECT ${columns.join(', ')} FROM ${entityId}`
+  }
+
+  const queryBundleRequest: QueryBundleRequest = {
+    concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
+    entityId,
+    partMask:
+      SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
+      SynapseConstants.BUNDLE_MASK_QUERY_RESULTS,
+    query: { sql },
+  }
+  const queryResultBundle = useGetQueryResultBundle(queryBundleRequest, {
+    enabled: shouldRun,
+  })
+
+  if (!shouldRun || error || !queryResultBundle?.data) {
+    return { data, error }
+  }
+
+  const colIndexes = columns.map(column => ({
+    column,
+    index: getFieldIndex(column, queryResultBundle.data),
+  }))
+
+  data =
+    queryResultBundle?.data?.queryResult?.queryResults.rows.map(el => {
+      const values = el.values as string[]
+      if (values.some(value => value === null)) {
+        console.warn('Row has null value(s) when no nulls expected')
+      }
+      const result = {}
+      colIndexes.forEach(({ column, index }) => {
+        result[column] = values[index]
+      })
+      return result
+    }) ?? []
+
+  return { data, error }
+}
 
-/**
- * Card view of challenges for the home page
- */
 export function ChallengesCardDeck() {
-  return <CardDeck cards={CHALLENGE_CARDS} cardDeckType="b2ai" />
+  const { data: challengesData = [], error: challengesError } = useTableFetch({
+    entityId: challengesTableId,
+    columns: [
+      CHALLENGES_TABLE_COLUMN_NAMES.ORG_ID,
+      CHALLENGES_TABLE_COLUMN_NAMES.IMG_HANDLE_ID,
+    ],
+  })
+
+  const {
+    data: challengesEntity,
+    error: challengesEntityError,
+    isLoading,
+  } = useGetEntity(challengesTableId)
+
+  const gcOrgIds = challengesData
+    .map(c => c[CHALLENGES_TABLE_COLUMN_NAMES.ORG_ID])
+    .map(id => `'${id}'`)
+    .join(', ')
+
+  const orgCols = [
+    ORG_TABLE_COLUMN_NAMES.ID,
+    ORG_TABLE_COLUMN_NAMES.NAME,
+    ORG_TABLE_COLUMN_NAMES.DESCRIPTION,
+  ]
+  const { data: gcOrgData, error: gcOrgError } = useTableFetch({
+    entityId: organizationTableId,
+    columns: orgCols,
+    sql: `SELECT ${orgCols.join(
+      ', ',
+    )} FROM ${organizationTableId} WHERE id IN (${gcOrgIds})`,
+    shouldRun: !!challengesData?.length,
+  })
+
+  if (challengesEntityError) {
+    return <ErrorBanner error={challengesEntityError} />
+  }
+  if (challengesError) {
+    return <ErrorBanner error={challengesError} />
+  }
+  if (gcOrgError) {
+    return <ErrorBanner error={gcOrgError} />
+  }
+  if (isLoading) {
+    return <div style={{}}>Loading Challenges entity</div>
+  }
+  if (isEmpty(challengesData)) {
+    return <div style={{}}>Loading Challenges table</div>
+  }
+  if (isEmpty(gcOrgData)) {
+    return <div style={{}}>Loading org data for GCs</div>
+  }
```

Comment:
End user does not care about what's loading -- ideally, this would render a Skeleton
```suggestion
  const isLoading = isLoadingChallengesTableQuery || isLoadingChallengesEntity  || isLoadingOrgsTableQuery
  if (isLoading) {
    return <div>Loading...</div>
  }
```